src/components/AdminSettings.vue

```diff
@@ -307,10 +490,27 @@ export default {
 			const htmlLink = `<a class="link" href="${this.adminFileStorageHref}" target="_blank" title="${linkText}">${linkText}</a>`
 			return t('integration_openproject', 'Copy the following values back into the OpenProject {htmlLink} as an Administrator.', { htmlLink }, null, { escape: false, sanitize: false })
 		},
+		userAppPasswordHint() {
+			const linkText = t('integration_openproject', 'Administration > File storages')
+			const htmlLink = `<a class="link" href="${this.adminFileStorageHref}" target="_blank" title="${linkText}">${linkText}</a>`
+			return t('integration_openproject', 'This value will not be accessible again after you clicking save. Copy this password to OpenProject {htmlLink} as an Administrator.', { htmlLink }, null, { escape: false, sanitize: false })
```

Comment:
Though in figma it's `clicking` I think grammatically this should be `click` I've commented in Figma as well for this
```suggestion
			return t('integration_openproject', 'This value will not be accessible again after you click save. Copy this password to OpenProject {htmlLink} as an Administrator.', { htmlLink }, null, { escape: false, sanitize: false })
```